src/azure/azure-mesh-adapter.ts

```diff
@@ -24,21 +25,68 @@ export class AzureMeshAdapter implements MeshAdapter {
     tenants: MeshTenant[],
     startDate: Date,
     endDate: Date,
+  ): Promise<void> {
+    const config = loadConfig();
+    if (config.azure.parentManagementGroups.length == 0) {
+      log.warning(
```

Comment:
I would not make this a "Warning" situation, but rather an info: "Hey, it seems you're not using Azure management groups. Did you know you can make your cost query a lot faster with using management groups? Do XYZ to know more".

Reply:
adapted the text